app/controllers/IneligibleController.scala

```diff
@@ -62,4 +63,9 @@ class IneligibleController @Inject() (
   val alreadyHaveAPaymentPlanPage: Action[AnyContent] = as.authenticatedJourneyAction { implicit request =>
     Ok(views.partials.ineligibleTemplatePage(Messages.NotEligible.`You already have a payment plan with HMRC`, views.partials.existingPaymentPlanPartial()))
   }
+
+  val cannotSetupDirectDebitOnlinePage: Action[AnyContent] = as.eligibleJourneyAction { implicit request =>
```

Comment:
not sure if this belongs in the `ineligibleController` - they're eligible according to the rules but they just can't set up a direct debit with the bank details given. Maybe this could go in the bank details controller?

Reply:
good point, I'll move it

Reply:
done